src/components/XChangeButton.js

```diff
@@ -7,11 +7,11 @@ const s = {
   fontStyle: "italic",
 };
 
-function XChangeButton(txt = "EMPTY", path = "", styl = s, clr = "#E0D03B") {
+function XChangeButton(txt = "EMPTY", path = "", styl = s) {
   return (
     <Link to={path}>
       <div class="button">
-        <Button sx={{ backgroundColor: clr }} style={{ width: "100%" }}>
```

Comment:
wait can this just be kept? in case we want to customize the button colour to be something else, like gray?

Reply:
oh my bad i didnt mean to delete that